doc/BBCode.md

```diff
@@ -1,208 +1,611 @@
 Friendica BBCode tags reference
 ========================
 
-* [Home](help)
-
-Inline
------
-
-
-<pre>[b]bold[/b]</pre> : <strong>bold</strong>
-
-<pre>[i]italic[/i]</pre> : <em>italic</em>
```

Comment:
Yes, it is outside of this PR. This PR just happened to get my attention to the quirk ;-)